src/test/java/collab/EmployeeManagerTest.java

```diff
@@ -118,11 +121,15 @@ public void executeCommandTest() throws Exception{
         EmployeeManager employeeManager = new EmployeeManager();
         List<ICommand> commandList = employeeManager.parseCommandList(commandStringList1);
         String executionResult = employeeManager.executeCommandList(commandList);
-        String[] resultLines = executionResult.split("\n");
+        assertEquals(executionResult, "SCH,NONE" + System.lineSeparator() +
+                "MOD,18117906,TWU QSOLT,CL4,010-6672-7186,20030413,PRO" + System.lineSeparator() +
+                "DEL,18117906,TWU QSOLT,CL3,010-6672-7186,20030413,PRO"  + System.lineSeparator());
+
+        String[] resultLines = executionResult.split(System.lineSeparator());
         assertTrue(resultLines.length == 3);
-        resultLines[0].equals("SCH,18117906,TWU QSOLT,CL4,010-6672-7186,20030413,PRO");
-        resultLines[1].equals("MOD,18117906,TWU QSOLT,CL3,010-6672-7186,20030413,PRO");
-        resultLines[2].equals("DEL,18117906,TWU QSOLT,CL3,010-6672-7186,20030413,PRO");
+        assertEquals(resultLines[0].trim(), "SCH,NONE");
+        assertEquals(resultLines[1].trim(), "MOD,18117906,TWU QSOLT,CL4,010-6672-7186,20030413,PRO");
+        assertEquals(resultLines[2].trim(), "DEL,18117906,TWU QSOLT,CL3,010-6672-7186,20030413,PRO");
```

Comment:
assertEquals라서 기대값이 첫번째 인자에 오면 좋을 것 같습니다.
```suggestion
        assertEquals("SCH,NONE", resultLines[0].trim());
        assertEquals("MOD,18117906,TWU QSOLT,CL4,010-6672-7186,20030413,PRO", resultLines[1].trim());
        assertEquals("DEL,18117906,TWU QSOLT,CL3,010-6672-7186,20030413,PRO", resultLines[2].trim());
```